plugins/package-managers/node/src/main/kotlin/Yarn.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.plugins.packagemanagers.node
 
 import java.io.File
+import java.lang.invoke.MethodHandles
```

Comment:
Commit message:
- "because `npm`" -> "because Yarn"
- "[1] https://github.com/oss-review-toolkit/ort/commit/ad9a3639604aacfa6aa3fd831593201f42e78483" -> "[1]: https://github.com/oss-review-toolkit/ort/commit/ad9a3639604aacfa6aa3fd831593201f42e78483" (colon after the footnote)